util/make_dir/make.nci

```diff
@@ -26,15 +25,19 @@ ARFLAGS     = -ruv
 # -g is necessary in F90FLAGS and LDFLAGS for pgf90 versions lower than 6.1
 # For compiling in double precision, put -r8
 # For compiling in single precision, remove -r8 and add -Duse_realtype_single
+NCI_INTEL_FLAGS = -r8 -i4 -traceback -fpe0 -convert big_endian -fno-alias -ip -check noarg_temp_created
+NCI_REPRO_FLAGS = -fp-model precise -fp-model source -align all
 ifeq ($(DEBUG), yes)
-    F90FLAGS_1  = -convert big_endian -i4 -r8 -traceback -O -debug all -g -check all -fpe0 -CB -fno-alias -ip -align all -assume nobuffered_io -check noarg_temp_created
-    CPPDEF    = -Duse_netCDF -Duse_comm_$(CHAN) -DTREAT_OVERLAY -DDEBUG -D__VERBOSE
-    MCT_FCFLAGS = -O0 -debug all -g -check all -fpe0 -xHost -ip -align all
+    NCI_DEBUG_FLAGS = -g3 -O0 -debug all -check all
+    F90FLAGS_1      = $(NCI_INTEL_FLAGS) $(NCI_REPRO_FLAGS) $(NCI_DEBUG_FLAGS) -CB -no-vec -assume nobuffered_io
+    CPPDEF          = -Duse_netCDF -Duse_comm_$(CHAN) -DTREAT_OVERLAY -DDEBUG -D__VERBOSE
+    MCT_FCFLAGS     = $(NCI_REPRO_FLAGS) $(NCI_DEBUG_FLAGS) -ip -fpe0 -xHost
```

Comment:
should this use `-xCORE-AVX2` instead of `-xHost` so we can run on Broadwell?

Reply:
I'm guessing `-xHost` implies `-axCORE-AVX512` if compiled on the Cascade Lake head nodes, which wouldn't run on Broadwell since that only supports `-axCORE-AVX2`...

Reply:
OK.

Reply:
The reason that I had left the flag on that line as `-xHost` is that I only tested with `DEBUG` off. I will need to re-test with `DEBUG=yes`. Also, if `DEBUG=yes`, then the flags include `-no-vec`. I will also double check what is done for the other flag files.

Reply:
I will also need to check where `FFLAGS` and `MCT_FCFLAGS` are used, since the `oasis3-mct` apparently uses both.

Reply:
Addressed by commit a8e4f9b